services/app-api/src/postgres/healthPlanPackage/insertHealthPlanPackage.ts

```diff
@@ -18,9 +18,9 @@ import { PopulationCoveredType } from '../../gen/gqlServer'
 
 export type InsertHealthPlanPackageArgsType = {
     stateCode: string
-    populationCovered?: PopulationCoveredType
+    populationCovered: PopulationCoveredType
     programIDs: string[]
```

Comment:
I think these fields are still optional until we remove that CHIP_ONLY_FORM feature flag